app.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const app = express();
+const {getTopics} = require ("/home/srvadmin/northcoders/backend/be-nc-news/contorllors/getTopics");
+
+
+
+app.use(express.json());
```

Comment:
This doesn't need to be here currently as it is not used for this endpoint. I appreciate it is probably to stop you forgetting it later but PRs should only have code for that ticket.